src/test/java/CowTest.java

```diff
@@ -0,0 +1,23 @@
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+public class CowTest {
+    Cow cow;
+
+    @Before
+    public void setUp() throws Exception {
+        cow = new Cow();
+    }
+
+    @Test
+    public void TestSound(){
+        assertEquals("MOOOOOOOOOOOOOO", cow.sound());
+    }
+
+    @Test
+
+
```

Comment:
Great job, but if you could change your spacing so that the lines are not too spread apart, would be great.